README.md

```diff
@@ -11,38 +13,67 @@ A python package for music and audio analysis.
 [![codecov](https://codecov.io/gh/librosa/librosa/branch/main/graph/badge.svg?token=ULWnUHaIJC)](https://codecov.io/gh/librosa/librosa)
 [![Docs](https://github.com/librosa/librosa/actions/workflows/docs.yml/badge.svg)](https://github.com/librosa/librosa/actions/workflows/docs.yml)
 
-Documentation
--------------
+ ##  Table of Contents
+
+**[Documentation](#Documentation)**<br>
+**[Installation](#Installation)**<br>
+
+- **[Using PyPI](#using-pypi)** <br>
+- **[Using Anaconda](#using-anaconda)** <br>
+- **[Building From Source](#building-from-source)** <br>
+- **[Hints for Installation](#hints-for-the-installation)** <br>
+
+**[Discussion](#Discussion)**<br>
+**[Citing](#Citing)**
+
+
+## Documentation
+
+
 See https://librosa.org/doc/ for a complete reference manual and introductory tutorials.
 
 The [advanced example gallery](https://librosa.org/doc/latest/advanced.html) should give you a quick sense of the kinds
 of things that librosa can do.
 
-Installation
-------------
+## Installation
+
+
+### - Using PyPI
 
 The latest stable release is available on PyPI, and you can install it by saying
 ```
 pip install librosa
 ```
 
-Anaconda users can install using ``conda-forge``:
+### - Using Anaconda
+
+Anaconda users can install using ```conda-forge```:
 ```
 conda install -c conda-forge librosa
 ```
 
-To build librosa from source, say `python setup.py build`.
-Then, to install librosa, say `python setup.py install`.
+### - Building from source
+
+To build librosa from source, say 
+```
+python setup.py build
+```
+Then, to install librosa, say 
+```
+python setup.py install
+```
 If all went well, you should be able to execute the demo scripts under `examples/`
```

Comment:
This line should be removed, as we removed the examples folder in the last release.  Suggested change:

> If all went well, you should be able to execute the following commands from a python console:

```python
>>> import librosa
>>> librosa.show_versions()
```
This should print out a description of your software environment, along with the installed versions of other packages used by librosa.